shared/test/dataDriver_test.js

```diff
@@ -29,7 +29,7 @@ describe("loop.DataDriver", () => {
     };
     sandbox.stub(window, "EventSource", url => {
       requests.push({ url });
-      return { addEventListener: () => {} };
+      return { addEventListener: () => {}, removeEventListener: () => {} };
```

Comment:
nit: let's split this onto multiple lines and just make them methods instead of keyed functions:
```js
{
  addEventListener() {},
  removeEventListener() {}
}
```